src/Plugins/LevelDBStore/IO/Data/LevelDB/Native.cs

```diff
@@ -82,155 +82,155 @@ public static class Native
         #endregion
 
         #region Env
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         public static extern IntPtr leveldb_create_default_env();
 
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         public static extern void leveldb_env_destroy(IntPtr /*Env*/ cache);
         #endregion
 
         #region Iterator
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         public static extern void leveldb_iter_destroy(IntPtr /*Iterator*/ iterator);
 
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         [return: MarshalAs(UnmanagedType.U1)]
         public static extern bool leveldb_iter_valid(IntPtr /*Iterator*/ iterator);
 
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
         public static extern void leveldb_iter_seek_to_first(IntPtr /*Iterator*/ iterator);
 
-        [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport("leveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
```

Comment:
Why it  changed?

Reply:
MacOS this is the name it should be, it will automatically pick up `libleveldb` if it exists of the other doesnt. So this way it finds the file on the OS.

Reply:
The name must be the same as the dll

Reply:
Not in `dotnet` it search for the name I listed above.

Reply:
but in windows is `libleveldb.dll` isn't it?